core/app/models/spree/order_updater.rb

```diff
@@ -73,14 +73,30 @@ def update_order_total
 
     def update_adjustment_total
       recalculate_adjustments
-      order.adjustment_total = line_items.sum(:adjustment_total) +
-                               shipments.sum(:adjustment_total)  +
+      lis = line_items.pluck(:adjustment_total, :included_tax_total, :additional_tax_total, :promo_total).reduce([0,0,0,0]) { |accum, item|
```

Comment:
Line is too long. [139/80]<br>Avoid using {...} for multi-line blocks.<br>Prefer `inject` over `reduce`.<br>Space missing after comma.